prime-router/settings/STLTs/PA/pa-phd.yml

```diff
@@ -0,0 +1,170 @@
+- name: "pa-phd"
+  description: "Pennsylvania Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "PA"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "pa-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation:
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/PA/PA-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "PA-ELR"
+        receivingApplicationOID: null
+        receivingFacilityName: "PADOH"
+        receivingFacilityOID: null
+        messageProfileId: null
+        replaceValue: null
+        replaceValueAwithB: null
+        reportingFacilityName: null
+        reportingFacilityId: null
+        reportingFacilityIdType: null
+        suppressQstForAoe: false
+        suppressHl7Fields: null
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: false
+        truncateHl7Fields: null
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides:
```

Comment:
Are we going to implement this valueSetOverrides in receiver transform?

Reply:
I asked Victor and he said these were done here.

Reply:
Fixed and tested, ready for re-review @oslynn